packages/kotti-ui/source/kotti-filters/components/FilterRow.vue

```diff
@@ -127,14 +123,20 @@ export default defineComponent<{
 			() => operationOptions.value.length <= 1,
 		)
 		const isValueFieldVisible = computed(
-			() => !isEmptyOperation(props.filter.operation, props.column?.type),
+			() =>
+				props.column &&
+				props.filter &&
+				!isEmptyOperation(props.filter.operation, props.column?.type),
```

Comment:
you added `props.filter &&` but it can just be `!isEmptyOperation(props.filter?.operation` --

stylistic difference but we usually prefer the `optional chaning` over explicit conditions

unless `isEmptyOperations` can accept a `null` - in which case it's correct as is

Reply:
actually on second thought

`isEmptyOperation` technically must be accepting `null` as a first argument because it used to before (props.filter.operation) had a default `null` value

so you can get away with `props.filter?.operation ?? null` as the first argument and remove the explicit check


```
() => !isEmptyOperation(props.filter?.operation ?? null, props.column?.type)
```

Reply:
@carsoli I believe this changes were introduced by @santiagoballadares
`isEmptyOperation` does not accept null for any of the arguments.

I will leave it as it was with the `optional.chaining` syntax.

Reply:
Actually `ts` complains in the second argument with the possibility of being undefined. 
Thats why we need the explicit check. So I will leave both explicit checks and remove the `optional.chaining`